head/src/test/kotlin/io/qalipsis/core/head/jdbc/repository/DataSeriesRepositoryIntegrationTest.kt

```diff
@@ -162,6 +164,50 @@ internal class DataSeriesRepositoryIntegrationTest : PostgresqlTemplateTest() {
         assertThat(dataSeriesRepository.findAll().count()).isEqualTo(1)
     }
 
+    @Test
+    fun `should not save data-series with color opacity above 100`() = testDispatcherProvider.run {
+        // given
+        val tenant = tenantRepository.save(tenantPrototype.copy())
+        val creator = userRepository.save(userPrototype.copy())
+
+        // then
+        assertThrows<DataAccessException> {
+            dataSeriesRepository.save(
+                DataSeriesEntity(
+                    reference = "other-series",
+                    tenantId = tenant.id,
+                    creatorId = creator.id,
+                    displayName = "my-name",
+                    dataType = DataType.METERS,
+                    colorOpacity = 101
+                )
+            )
+        }
+
+    }
+
+    @Test
+    fun `should not save data-series with color opacity below 1`() = testDispatcherProvider.run {
```

Comment:
Please disable that test until we understand why the validation is not triggered here.